### PR TITLE
Feature/update pairing isactive property

### PR DESCRIPTION
**Pull request by TalhaAli00**

### CODE REVIEW

File: core/android/src/main/kotlin/com/walletconnect/android/internal/common/model/Pairing.kt [hunk 1]
```diff
@@ -12,17 +12,17 @@ data class Pairing(
     val relayProtocol: String,
     val relayData: String?,
     val uri: String,
-    val isActive: Boolean,
     val registeredMethods: String
 ) : Sequence {
+    val isActive: Boolean
+        get() = (expiry.seconds > CURRENT_TIME_IN_SECONDS)
```
Comment (jakubuid) [inline core/android/src/main/kotlin/com/walletconnect/android/internal/common/model/Pairing.kt:18]: I'm not sure if we want to do this. Can you check such flow:
- open QR code on a dapp
- scan with a wallet
- dismiss proposal dialog on the wallet (do not approve nor reject)
- scan the same qr code
- is proposal dialog shown?

Comment (TalhaAli00) [reply core/android/src/main/kotlin/com/walletconnect/android/internal/common/model/Pairing.kt:18]: @jakubuid the wallet is showing that the pairing already exists. Is that expected?

Comment (jakubuid) [reply core/android/src/main/kotlin/com/walletconnect/android/internal/common/model/Pairing.kt:18]: We have regression with this change, I tested the flow above and users cannot scan the same QR code twice

Comment (TalhaAli00) [reply core/android/src/main/kotlin/com/walletconnect/android/internal/common/model/Pairing.kt]: Updated the logic and the above test case works now

**Review by jakubuid — APPROVED** [review]

**Review by Elyniss — APPROVED** [review]

**Review by kacperoak — APPROVED** [review]

**Pull request merged**